### PR TITLE
Data model

### PR DESCRIPTION
**Pull request by ezraermy**
In this pull request, I:
-  Created an app and created a Postgres database setup.
- Built project schema.
- Created and run the necessary migration files.
- Table and column names are according to the given ERD diagram.
#### Other requirements
- No linter errors
- Used correct gitflow
- Documented in a professional way

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      t.datetime "updated_at", null: false
+      t.index ["author_id"], name: "index_likes_on_author_id"
+      t.index ["post_id"], name: "index_likes_on_post_id"
+    end
+  
+    create_table "posts", force: :cascade do |t|
+      t.string "title"
+      t.text "content"
+      t.integer "commentsCounter"
+      t.integer "likesCounter"
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] We should maintain the ERD diagram recommendation. We should have "text", not "content". Kindly rename that column.

- [ ] According to ruby practices, our table names should follow the snake case convention. Kindly rename `commentsCounter` and `likesCounter` to `comments_counter` and `likes_counter` respectively.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      t.datetime "created_at", null: false
+      t.datetime "updated_at", null: false
+      t.index ["author_id"], name: "index_posts_on_author_id"
+    end
+  
+    create_table "users", force: :cascade do |t|
+      t.string "name"
+      t.string "photo"
+      t.text "bio"
+      t.integer "postCounter"
```
Comment (julie-ify) [inline db/schema.rb]: - [ ] Kindly rename this column to `posts_counter`.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @ezraermy,

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/schema.rb (hunk 2 above)
Comment (ezraermy) [reply db/schema.rb]: Thanks a lot, @julie-ify for your feedback.

**Review by VICTOR-CHUKWU — APPROVED** [review]
Hi @ezraermy , 
# STATUS: APPROVED! 🥇 💯 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linters passed

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
Hi @VICTOR-CHUKWU, Thank you for your kind effort to review my code.

**Pull request merged**